hoomd/md/pair/pair.py

```diff
@@ -1157,6 +1157,80 @@ def __init__(self, nlist, default_r_cut=None, default_r_on=0., mode='none'):
         self._add_typeparam(params)
 
 
+class ExpandedMie(Pair):
+    """Expanded Mie pair potential.
+
+    Args:
+        nlist (`hoomd.md.nlist.NList`): Neighbor list
+        default_r_cut (float): Default cutoff radius :math:`[\\mathrm{length}]`.
+        defult_r_on (float): Default turn-on radius :math:`[\\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    `ExpandedMie` specifies that a radially shifted Mie pair potential should be
+    applied between every non-excluded particle pair in the simulation.
+
+    .. math::
+        :nowrap:
+
+        \\begin{eqnarray*}
+        V_{\\mathrm{mie}}(r)
+          = & \\left( \\frac{n}{n-m} \\right) {\\left( \\frac{n}{m}
+          \\right)}^{\\frac{m}{n-m}} \\varepsilon \\left[ \\left(
+          \\frac{\\sigma}{r} \\right)^{n} - \\left( \\frac{\\sigma}{r}
+          \\right)^{m} \\right] & r < r_{\\mathrm{cut}} \\\\
+          = & 0 & r \\ge r_{\\mathrm{cut}} \\\\
```

Comment:
```suggestion
          = & \\left( \\frac{n}{n-m} \\right) {\\left( \\frac{n}{m}
          \\right)}^{\\frac{m}{n-m}} \\varepsilon \\left[ \\left(
          \\frac{\\sigma}{r-\Delta} \\right)^{n} - \\left( \\frac{\\sigma}{r-\Delta}
          \\right)^{m} \\right] & r < r_{\\mathrm{cut}} \\\\
          = & 0 & r \\ge r_{\\mathrm{cut}} \\\\
```
Missing the expanded part in the function.